src/ORM/Filters/SearchFilter.php

```diff
@@ -437,7 +463,7 @@ public function isEmpty()
     /**
      * Determines case sensitivity based on {@link getModifiers()}.
      *
-     * @return Mixed TRUE or FALSE to enforce sensitivity, NULL to use field collation.
+     * @return ?bool TRUE or FALSE to enforce sensitivity, NULL to use field collation.
```

Comment:
I haven't actually changed the return type - it was always returning only these types.